users/test.py

```diff
@@ -20,17 +20,43 @@
 """
 
 import unittest
+import mongomock
 from app import is_authorized_to_edit
 
+
 class TestAuthorization(unittest.TestCase):
-    """Test /authorization endpoint of users service"""
-    def test_good_user_is_authorized(self):
-        """An authorized user should have authorized privileges"""
-        self.assertTrue(is_authorized_to_edit("carolyn", None))
-
-    def test_bad_user_not_authorized(self):
-        """An non-authorized user should not have authorized privileges"""
-        self.assertFalse(is_authorized_to_edit("Voldemort", None))
+    """Test /authorization endpoint of users service."""
+
+    def setUp(self):
+        """Seed mock DB for testing"""
+        self.id_of_test_user_is_authorized = "authorized-user"
```

Comment:
For these sorts of constants, a global is usually fine, e.g. at the top level:

AUTHORIZED_USER = "authorized-user"
INTRUDER = "non-authorized-user"
MISSING = "not-a-user-in-the-database"

Reply:
Done